### PR TITLE
[zh] Migrating telemetry and security agents from dockershim

### PR DESCRIPTION
**Pull request by X723166587**
Apply PR #31607 corrections to Chinese content.

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:287]:
```suggestion
不要为 `docker-container-stats` 监视器使用 Docker Engine 之外的运行时。
```

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:295]: 删除此空行，留一个

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:300]: 删除此空行，留一个

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:314]: 删除此空行，留一个

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:310]: 删除空行310

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:282]: 删除此空行

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:250]: 删除此空行

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:233]: 删除此空行

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:213]: 删除此空行

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:189]: 删除此空行

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:175]: 删除此空行

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:53]: 删除此空行

**Review by tengqm — COMMENTED** [review]
please squash your commits.

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (Sea-n) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:301]:
```suggestion
Flame 不支持 Docker 以外的容器运行时，具体可见 [https://github.com/yahoo/kubectl-flame/issues/51](https://github.com/yahoo/kubectl-flame/issues/51)
```

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (Sea-n) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:278]:
```suggestion
`kubelet-stats` 监视器此前已被弃用，现支持 `kubelet-metrics`。
```

**Review by Sea-n — COMMENTED** [review]
Looks good to me, just a few nitpick.

**Comment by tengqm** [comment]
/approve
/hold
Good to go once commits are squashed.

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (howieyuen) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:278]:
```suggestion
`kubelet-stats` 监视器此前已被供应商所弃用，现支持 `kubelet-metrics`。
```

File: content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
Comment (howieyuen) [inline content/zh-cn/docs/tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md:289]:
```suggestion
收集的指标会发生变化。具体请查看你的告警规则和仪表盘。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

**Comment by Sea-n** [comment]
Hey @X723166587,

Please squash your changes into one commit.
You need to download local git client to do it.

**Review by howieyuen — APPROVED** [review]
/lgtm

**Pull request merged**
